frontend/src/components/common/Pagination.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useEffect, useState } from "react";
+import { extendTheme, chakra, ChakraProvider, Box } from "@chakra-ui/react";
+import {
+  Pagination as P,
+  usePagination,
+  PaginationNext,
+  PaginationPage,
+  PaginationPrevious,
+  PaginationContainer,
+  PaginationPageGroup,
+  PaginationSeparator,
+} from "@ajna/pagination";
+
+const customTheme = {
+  ...chakra,
+  fonts: {
+    body: '"DM Sans"',
+    heading: '"DM Sans"',
+    mono: '"DM Sans"',
+  },
+  colors: {
+    brand: {
+      900: "#154472",
+    },
+  },
+  components: {
+    PaginationSeparator: {
```

Comment:
we already have a custom theme! can you move this into `themes/components/paginationSeparator.ts`

Reply:
you will also need to update `themes.index.ts`